templates/legal/ubuntu-advantage-service-description/ja/_base_legal_markdown_ja.html

```diff
@@ -0,0 +1,26 @@
+{% extends "templates/base.html" %}
+{% load versioned_static  %}
+{% block title %}{{ title }}{% endblock title %}
+{% block meta_description %}{{ description }}{% endblock meta_description %}
```

Comment:
Question: should the meta description be translated? At the moment it's in English.